src/common/function/FunctionManager.cpp

```diff
@@ -577,10 +582,78 @@ FunctionManager::FunctionManager() {
         }
         case Value::Type::INT:
         case Value::Type::FLOAT: {
-          if (args.size() == 2) {
+          if (args.size() >= 2) {
             if (args[1].get().type() == Value::Type::INT) {
+              auto val = args[0].get().getFloat();
```

Comment:
Add test case: `round(3, 2)`

Reply:
Is args[0] always float type here?

Reply:
The current implementation of `master` branch assumes that args[0] is float and does not do anything if it is int

Reply:
`getFloat()` will crash if it is int type.

Reply:
@Koukyosyumei

Reply:
The case condition ensures that it's float.

Reply:
ok